src/styles/Signup.css

```diff
@@ -0,0 +1,79 @@
+.Account {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 100vh;
+  min-width: 700px;
+}
+
+.Account-Content {
+  display: flex;
+  height: calc(100% - 100px);
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.Account-Panel {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  text-align: left;
+  background: white;
+  box-shadow: 4px 4px 10px #212121;
+  width: 600px;
+  height: 1360px;
+  border-radius: 20px;
+  padding: 60px;
+  margin: 100px;
+}
+
+.Account-Title {
+  font-family: "ABeeZee";
+  color: #1f1f1f;
+  font-size: 50px;
+  font-weight: normal;
+  text-align: center;
+}
+
+.Account-Input-Container {
+  margin: 25px 0px;
+}
+
+.Account-Label-Text {
+  font-family: "ABeeZee";
+  color: #5b5b5b;
+  font-size: 24px;
+}
+
+.Account-Sub-Label {
+  margin-top: 20px;
+  margin-bottom: 15px;
+  font-family: "ABeeZee";
+  color: #1f1f1f;
+  font-weight: normal;
+  font-size: 30px;
+}
+
+.Account-Input {
+  width: 100%;
+  height: 50px;
+  line-height: 40px;
+  padding-left: 15px;
+  font-family: "ABeeZee";
+  font-size: 20px;
+  border-color: #969696;
+  color: #1f1f1f;
+  margin-bottom: 15px;
+  border-radius: 10px;
+}
+
+.Account-Button {
+  height: 55px;
+  width: 100%;
+  text-align: center;
+  box-shadow: 0px 0px 0px 0px;
+  font-size: 24px;
+  margin-top: 30px;
+  border-radius: 10px;
```

Comment:
If you're adding this styling here, then the Login.css login button also needs a border-radius of 10px